project.yaml

```diff
@@ -5,8 +5,10 @@ expectations:
   population_size: 1000
 
 actions:
-  generate_dataset:
-    run: ehrql:v1 generate-dataset analysis/dataset_definition.py --output output/dataset.csv.gz
+  generate_pf_measures:
```

Comment:
We should discuss if we want one measures definition for everything, or multiple measure definitions, e.g., one for pharmacy first codes, one for pharmacy first conditions.